kod/object/active/holder/nomoveon/battler/player.kod

```diff
@@ -3227,7 +3227,7 @@ messages:
 
       if NOT Send(Send(SYS,@GetParliament),@BetaPotionsEnabled)
       {
-         iCost = Bound(piMax_mana/2+1,11,$);
+         iCost = piMana * Send(Send(SYS,@GetSettings),@GetBroadcastManaCost) / 100;
```

Comment:
You removed a call to Bound here that previously set the minimum mana required to broadcast to 11.  Without that, suppose piMana is 1.  Then as long as the setting is < 100, the mana cost is 0. Oops...

My suggestion is to put the minimum bound on, and possibly add a special case for when the setting is 0 that bypasses the bound.  Thus an admin could make broadcasts free if desired.

Reply:
done (including using piMax_Mana and not piMana to base the cost on, which was not intended).  I also set the default penalty percent to 8% instead of 0 to start with.